### DISCUSSION TITLE
ValueError: The onnxruntime python package is not installed. Please install it with `pip install onnxruntime`

### DISCUSSION THREAD

**Discussion by Assassinator-ban** [open-webui/open-webui#6889] [General]
I created a virtual environment using python3.11.9 on the windows10 system and executedpip install open-webui，After running the startup command, I got an error like this:The onnxruntime python package is not installed. Please install it with `pip install onnxruntime`，
I try to executepip install onnxruntime，but it gave me this hint：(env-openwebui) PS T:\envs> pip install onnxruntime
Requirement already satisfied: onnxruntime in t:\envs\env-openwebui\lib\site-packages (1.20.0)
Requirement already satisfied: coloredlogs in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (15.0.1)
Requirement already satisfied: flatbuffers in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (24.3.25)
Requirement already satisfied: numpy>=1.21.6 in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (1.26.4)
Requirement already satisfied: packaging in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (23.2)
Requirement already satisfied: protobuf in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (4.25.5)
Requirement already satisfied: sympy in t:\envs\env-openwebui\lib\site-packages (from onnxruntime) (1.13.1)
Requirement already satisfied: humanfriendly>=9.1 in t:\envs\env-openwebui\lib\site-packages (from coloredlogs->onnxruntime) (10.0)
Requirement already satisfied: mpmath<1.4,>=1.1.0 in t:\envs\env-openwebui\lib\site-packages (from sympy->onnxruntime) (1.3.0)
Requirement already satisfied: pyreadline3 in t:\envs\env-openwebui\lib\site-packages (from humanfriendly>=9.1->coloredlogs->onnxruntime) (3.5.4)， It shows that I have installed it, but I cannot use the startup command to run it.webui
![Snipaste_2024-11-13_06-09-48](https://github.com/user-attachments/assets/9a46f722-d2c7-4979-a6c3-b0006a3c443b)

**Comment by wulfsystems** [comment]
Same issue but I use Win11.
I had to install the newest Microsoft Visual C++ Redistributable
https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Reply by ghost** [reply]
Thanks this helped me resolve the problem.

**Reply by zane797** [reply]
Seconded, helped resolve it on Windows 10 Pro 22H2

**Reply by hiyamgh** [reply]
Worked for me, thank you so much!

**Reply by sivasundharam** [reply]
Worked for me, thank you.

**Comment by LeoOliveira360** [comment]
Works, thank you.

**Comment by JT-Gresham** [comment]
This issue is not restricted to Windows. My installation is on Arch Linux and I'm seeing the same issue after a fresh install. I was using onnxruntime-openvino successfully on my previous installation. After checking my site-packages directory, I see onnxruntime is installed as it should be...and ollama runs in the terminal as it should on my Intel Arc A770.
